src/components/Cart/Cart.vue

```diff
@@ -75,24 +79,24 @@ export default {
   }
 
   .np-cart__arrow {
-    transform: rotateZ(180deg);
+    @apply transform rotate-180 left-2.5;
   }
 }
 
 .np-cart--opened {
   @apply bg-secondary-lighter w-1/4 h-screen flex flex-col items-center relative text-secondary-dark;
   .np-cart__arrow {
+    @apply left-3;
   }
 }
 
 .np-cart__arrow__container {
-  @apply h-12 w-12 rounded-full bg-primary absolute -left-6 flex justify-center items-center;
+  @apply h-12 w-12 rounded-full bg-primary absolute -left-6 flex justify-center items-center cursor-pointer;
   top: calc(50% - 1.5rem);
-  cursor: pointer;
 
   .np-cart__arrow {
```

Comment:
you have np-cart repeated multiple times, it is better to use one base `.np-cart` class and then nest more detailed classes inside np-cart block. Use `&` in this case, e.g 
```
.np-cart {
  &__arrow {
  }
}
```

Reply:
Tutaj zmiany wprowadziłem na innej gałęzi już dawno to poprawiłem, tylko nie chciałem mergować wszystkiego jak czekałem na pull requesty :)